signal/signal.go

```diff
@@ -109,6 +109,11 @@ func executeRunner(c config.Config) error {
 
 func Start() {
 	config, configErr := config.ParseArgsReturnConfig(os.Args[1:])
+	if configErr != nil {
+		for _, err := range configErr {
+			log.Error(err)
+		}
+	}
```

Comment:
If there is a configErr, shouldn't we `os.Exit(1)` here? As I read this currently, it logs that there was an error in the config and then continues.